ui/src/preview/preview-table-item.tsx

```diff
@@ -33,6 +33,7 @@ export type PreviewTableItemProps = {
     publicationGroupAmounts: PublicationGroupAmounts;
     displayedTotalPublicationAssetAmount: number;
     onShowOnMap: (bbox: BoundingBox) => void;
+    isValidating: (item: PreviewTableEntry) => boolean;
```

Comment:
Tästä eteenpäin tiketin side dishiä: Spinnerit näytetään nyt rivikohtaisesti validaation ajautuessa. Tämä muuttui sen myötä getter-funktioksi, sillä staged- ja unstaged-muutosten laatikoilla on eri ehdot spinnerin näyttämiselle.